openupgrade_scripts/scripts/l10n_vn/17.0.2.0.3/pre-migration.py

```diff
@@ -26,7 +26,8 @@ def _vn_coa_rename_xml_id(env):
     """
     env.cr.execute("""SELECT id FROM res_company WHERE chart_template = 'vn'""")
```

Comment:
@duong77476-viindoo sửa lại sql như này thì có thể dùng như cũ được
```suggestion
    env.cr.execute("""SELECT array_agg(id) FROM res_company WHERE chart_template = 'vn'""")
```

Reply:
Oki a